examples/basic/variables.tf

```diff
@@ -13,3 +13,9 @@ variable "existing_kms_instance_guid" {
   type        = string
   description = "GUID of an existing kms instance"
 }
+
+variable "force_delete" {
```

Comment:
don't expose this in the example - hard code it to `true` and leave a code comment saying its set to true for testing purposes